tests/hooks/useRecentSearches.test.tsx

```diff
@@ -0,0 +1,86 @@
+import { useRecentSearches } from '../../src/hooks/useRecentSearches';
+import { renderHook } from '@testing-library/react-hooks';
+
+it('returns recent searches', () => {
+  const verticalKey = 'people';
+  const { result, rerender } = renderHook(() =>
+    useRecentSearches(10, verticalKey)
+  );
+  const [, setRecentSearch, clearRecentSearches] = result.current;
+  setRecentSearch('bob');
+  setRecentSearch('carrie');
+  setRecentSearch('williard');
+
+  rerender();
+  const [recentSearches] = result.current;
+  expect(recentSearches?.length).toBe(3);
```

Comment:
let's also check the contents of the recent searches, not just that the length is 3. It's probably fine if this is the only test for the actual contents

Reply:
done!